cluster-addons/cluster-autoscalar/ca-iam-policy.tf

```diff
@@ -0,0 +1,59 @@
+resource "aws_iam_role_policy" "frankfurt_ca_policy" {
+  name = "frankfurt_ca_policy"
```

Comment:
Can we not name it `frankfurt`? It should be a generic one.

Reply:
done